clang-tools-extra/docs/clang-tidy/checks/readability/use-span-first-last.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - readability-use-span-first-last
+
+readability-use-span-first-last
+===============================
+
+Suggests using ``std::span::first()`` and ``std::span::last()`` member functions 
+instead of equivalent ``subspan()``. These dedicated methods were added to C++20 
+to provide more expressive alternatives to common subspan operations.
+
+Covered scenarios:
+
+=========================== ============
```

Comment:
Second column should be expanded on two characters to fit `s.first(n)` with all back-ticks.

Reply:
thank you, super sorry that this is such a back n forth, but my brain seems to be wired a way to not being able to figure RST out 🙈

Reply:
You just need to look on diff :-)